crate/azure_cvm/src/error.rs

```diff
@@ -0,0 +1,25 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum Error {
+    #[error("AkPub not found")]
+    AkPubNotFound,
+    #[error("BadURLError: {0}")]
+    BadURLError(String),
+    #[error("binary parse error")]
+    BinaryParseError(#[from] bincode::Error),
+    #[error("DecodeError: {0}")]
+    DecodeError(#[from] base64::DecodeError),
+    #[error("ImdsReponseError: {0}")]
+    ImdsReponseError(String),
```

Comment:
```suggestion
    #[error("ImdsResponseError: {0}")]
    ImdsResponseError(String),
```

Reply:
Fixed in 4767391487ed7165d356b439f270d4b3cf1d3cdc.